### PR TITLE
Dark theme and monospace

### PR DESCRIPTION
**Pull request by paulauus**
Dark theme for the dashboard + chose monospace as font, because it seems most context-appropriate.

### CODE REVIEW

File: dashboard/.streamlit/config.toml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[theme]
+base = "dark"
+primaryColor = "#6a0dad"  # Purple primary color
+backgroundColor = "#121212" # Dark background color
```
Comment (Leon-JS02) [inline dashboard/.streamlit/config.toml:4]: Gamers love dark themes 😍

**Review by Leon-JS02 — APPROVED** [review]
A good initial start on the styling for the dashboard!

File: dashboard/.streamlit/config.toml (hunk 1 above)
Comment (MichaelHassan) [reply dashboard/.streamlit/config.toml:4]: Agreed. Well done catering to target audience.

**Review by MichaelHassan — APPROVED** [review]
Good start.

File: dashboard/.streamlit/config.toml (hunk 1 above)
Comment (mehrdadhalali) [inline dashboard/.streamlit/config.toml:4]: Lovely scheme

**Review by mehrdadhalali — APPROVED** [review]

**Pull request merged**